comp/ops/compbox.yaml

```diff
@@ -0,0 +1,34 @@
+summary: Setup the compboxes for the $SRYYYY competition
```

Comment:
I don't think these should be merged. While there are overlaps in the work to be done, setting up the public compbox is often completed ~months before the competition. These being on the same ticket would leave this task started, blocked & incomplete for a long time which is often hard to track.
I think it would be much clearer to have separate tasks for the public compbox than the in-venue one (as was previously the case).

Reply:
We've split these into two tasks.